onnx2kerastl/operation_layers.py

```diff
@@ -771,7 +783,7 @@ def convert_nms(node, params, layers, lambda_func, node_name, keras_name):
             batch_tensor = batch * tf.ones_like(indices)
             res = tf.stack([batch_tensor, class_tensor, indices], axis=-1)
             all_results.append(res)
-    layers[node_name] = tf.concat(all_results, axis=0)
+    layers[node_name] = tf.cast(tf.concat(all_results, axis=0), dtype=tf.int64)
```

Comment:
Do we work on int64 from now on? 
Do you remember why this was specifically needed?